go-controller/hack/update-codegen.sh

```diff
@@ -53,13 +53,15 @@ for crd in ${crds}; do
     --input-base "" \
     --input github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/$crd/v1 \
     --output-package github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/$crd/v1/apis/clientset \
+    --plural-exceptions="EgressQoS:EgressQoSes" \
```

Comment:
this is not needed for other CRDs ike egressIPs/egressFirewall... so maybe do this only if `crd` is egressQoS and add a comment on the code here on why we need the plural exception..
follow-up:
and did we figure out why we need this plural exception? is it because codegen doesn't like `QoSes` ?

Reply:
> this is not needed for other CRDs ike egressIPs/egressFirewall... so maybe do this only if `crd` is egressQoS and add a comment on the code here on why we need the plural exception.. 

I thought about it, but adding this flag doesn't affect the others and I think is cleaner than adding some conditional (+ can be extended later with a comma if more resources need a plural exception), wdyt?

>follow-up: and did we figure out why we need this plural exception? is it because codegen doesn't like `QoSes` ?

when I initially created EgressQoS I remember fighting with its plural and what solved it was adding the `// +kubebuilder:resource:path=egressqoses` tag (it could also be that I forgot what I did and used some hack :smile: ). not sure what happened, but seems using this flag is what we need to enforce it